osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerMatchSongSelect.cs

```diff
@@ -109,5 +110,9 @@ public override bool OnExiting(IScreen next)
         }
 
         protected override BeatmapDetailArea CreateBeatmapDetailArea() => new PlayBeatmapDetailArea();
+
+        protected override ModSelectOverlay CreateModSelectOverlay() => new ModSelectOverlay(isValidMod);
+
+        private bool isValidMod(Mod mod) => !(mod is ModAutoplay) && (mod as MultiMod)?.Mods.Any(mm => mm is ModAutoplay) != true;
```

Comment:
This should probably be in `MatchSongSelect` to avoid the same scenario for the playlists mode.